uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -378,6 +378,29 @@ _Response Fields_
   Added in UAA 3.3.0
 </aside>
 
+When an access token is revoked using this endpoint, the UAA Introspect Token endpoint (``/introspect``)
```

Comment:
> an access token is revoked using this endpoint

We are trying to document what it means to "revoke a token using this endpoint", so we should try to not use this phrasing in the documentation.

I'm thinking of something like

> When an access token is passed to the `/revoke` endpoint, and the UAA responds positively to the request by returning a 200 HTTP code, the same token then passed to the `/introspect` endpoint will respond with `active: false`. We will say that the token has been revoked.

Reply:
Point discussed and/or edits made.